src/main/resources/run-class.sh

```diff
@@ -98,9 +98,17 @@ if [ -e conf/env.conf ]; then
   CONF_FILE="-Dconfig.file=conf/env.conf"
 fi
 
+#Add a cache-key config, user tests for existence
+ADD_CACHE_KEY=""
```

Comment:
You have three variables, all with similar names: `ADD_CACHE_KEY`, `ADD_CACHEKEY`, and `ADDCACHEKEY`. For best results, use just one. :)